static/app/components/events/eventEntry.tsx

```diff
@@ -148,40 +154,34 @@ function EventEntry({
         />
       );
     case EntryType.SPANS:
-      return (
```

Comment:
Tbh, I'm unsure who was using this before... I believe it was added by the session replay team, but never ended up being used. I think it should be safe for us to just hijack it for perf issues instead